src/sickle.ts

```diff
@@ -223,6 +232,9 @@ class Annotator {
     if (/^\/\*\*/.test(comment) && /\*\/$/.test(comment)) {
       return comment.slice(3, comment.length - 2);
     }
+    if (/^\/\/\//.test(comment)) {
```

Comment:
Ahhh I was being too clever here -- I don't think closure obeys them either.  I reduced the test and delete this.